editor/editor_settings.cpp

```diff
@@ -573,6 +573,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("editors/2d/bone_outline_color", Color(0.35, 0.35, 0.35));
 	_initial_set("editors/2d/bone_outline_size", 2);
 	_initial_set("editors/2d/viewport_border_color", Color(0.4, 0.4, 1.0, 0.4));
+	_initial_set("editors/2d/constraint_editor_view", true);
```

Comment:
Should be "constrain_editor_view", never trust a Frenchman coming up with an English term :P

Reply:
Damn it, looks like someone is making fun of me...  :p. 
Sorry for the mistake @KoBeWi, we Frenchies tend to add letters where they are not needed.